Demo/Common/IntegrationExamples/Session/Components/BLIKComponentExample.swift

```diff
@@ -0,0 +1,110 @@
+//
+// Copyright (c) 2025 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import AdyenActions
+import AdyenCheckout
+import AdyenComponents
+
+internal final class BLIKComponentExample: InitialDataFlowProtocol {
+    
+    internal weak var presenter: PresenterExampleProtocol?
+    
+    private var adyenCheckout: AdyenCheckout?
+    private var adyenComponent: AdyenCheckoutComponent?
+    
+    internal lazy var apiClient = ApiClientHelper.generateApiClient()
+    
+    // comes from demo app protocol, unused on new structure
+    internal lazy var context: AdyenContext = generateContext()
+    
+    func start() {
+        presenter?.showLoadingIndicator()
+        
+        Task {
+            do {
+                let sessionResponse = try await requestSessionInitialInfo()
+                let component = try await self.blikComponent(from: sessionResponse)
+                self.adyenComponent = component
+                await hideLoading()
+                await present(component: component)
+            } catch {
+                await hideLoading()
+                await handleError(error)
+            }
+            
+        }
+    }
+    
+    private func blikComponent(from sessionResponse: SessionResponse) async throws -> AdyenCheckoutComponent {
+        
+        let configuration = try CheckoutConfiguration(
+            environment: ConfigurationConstants.componentsEnvironment,
+            amount: ConfigurationConstants.current.amount,
+            clientKey: ConfigurationConstants.clientKey,
+            analyticsConfiguration: .init(
+                isEnabled: ConfigurationConstants.current.analyticsSettings.isEnabled
+            )
+        ) {
+            BLIKComponentConfiguration()
+        }
+        .onComplete { [weak self] result in
+            self?.presenter?.dismiss {
```

Comment:
A helper method would make it easier to read. We can also think of presenting an alert first and then dismissing as alert completion?

Reply:
Sure good idea